mohawk/tests.py

```diff
@@ -142,29 +143,42 @@ def test_post_content_type_with_trailing_charset(self):
         self.receive(sn.request_header, method=method, content=content,
                      content_type='application/json; charset=other')
 
-    @raises(ValueError)
+    @raises(MissingContent)
     def test_missing_payload_details(self):
-        self.Sender(method='POST', content=None, content_type=None)
+        self.Sender(method='POST', content=EmptyValue,
+                    content_type=EmptyValue)
 
     def test_skip_payload_hashing(self):
         method = 'POST'
         content = '{"bar": "foobs"}'
         content_type = 'application/json'
-        sn = self.Sender(method=method, content=None, content_type=None,
+        sn = self.Sender(method=method, content=EmptyValue,
+                         content_type=EmptyValue,
                          always_hash_content=False)
         self.receive(sn.request_header, method=method, content=content,
                      content_type=content_type,
                      accept_untrusted_content=True)
 
-    @raises(ValueError)
+    def test_empty_payload_hashing(self):
+        method = 'GET'
+        content = None
+        content_type = None
+        sn = self.Sender(method=method, content=content,
+                         content_type=content_type)
+        self.assertTrue('hash="' in sn.request_header)
+        self.receive(sn.request_header, method=method, content=content,
+                     content_type=content_type,
+                     accept_untrusted_content=True)
```

Comment:
The thing about changing the default to `EmptyValue` is that `content=None` is now treated the same as `content=''`, so as not to trip on web frameworks that return `None` instead of empty strings, so an error isn't raised in that case anymore. (This is to differentiate between passing in empty content and failing to pass in content at all.)